### ISSUE TITLE
Implement the additional identified nemo variables

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#381]
See issue 595 at the ec-earth portal: it concerns Shaconemo dummy variables identified by the ec-earth consortium

**Comment by treerink** [comment]
Adding this additional identified nemo variables is proceeding in combination with the repeated identification update due CMIP6 data request version 01.00.29 #346. So these two issues go together.

**Comment by treerink** [comment]
This issue is about[ this list](https://docs.google.com/spreadsheets/d/189ClDiFx3urHpiuXuMnQKkkpIeV2P-2Zy11ffNnQ408/edit#gid=1234946926) of additional identified NEMO variables. Variables that are not identified in the Shaconemo ping file, but which have been identified by the ec-earth consortium (Raffaelle mainly). Three of these variables have been brought to the ignored list in 
9f0eb9a, because they will be only manually activated in OMIP.

**Comment by treerink** [comment]
This issue is about[ this list](https://docs.google.com/spreadsheets/d/189ClDiFx3urHpiuXuMnQKkkpIeV2P-2Zy11ffNnQ408/edit#gid=1234946926) of additional identified NEMO variables. Variables that are not identified in the Shaconemo ping file, but which have been identified by the ec-earth consortium (Raffaelle mainly). Three of these variables have been brought to the ignored list in , because they will be only manually activated in OMIP.

**Comment by treerink** [comment]
Ready for testing: run EC-Earth3 with these added variables and cmorize them.

**Comment by treerink** [comment]
Now `genecec` is also updated for the added NEMO variables, and with that the testing can start indeed.

**Comment by treerink** [comment]
Except of one issue with `zhalfo` (to be fixed in the control output files), I was able to run EC-Earth3 with these additional output. With two of these new variables we have cmorizization issues: #387 & #388. In total 176 variables are successful cmorized (before 171), so it seems some other new ones (in total 15 added ones) were not cmorized but did not cause a crash.

The point with `zhalfo` is that it is requested as Omon montly variable but also as Oclim variable. The latter needs also montly output: What happens now is that actually the same montly output is asked, and the effect is that two times a variable with the same variable name  `zhalfo` is written in the same netcdf file, which is not possible for netcdf. To temporarily circumvent this, I just removed the `zhalfo` `Oclim` from the `file_def` file I was using, but this needs a sound solution.
